cmd/exo/cmd/root.go

```diff
@@ -57,18 +72,15 @@ func buildClient() {
 		return
 	}
 
-	var err error
-	cs, err = client.BuildClient(configFilePath, region)
-	if err != nil {
-		log.Fatal(err)
-	}
+	cs = egoscale.NewClient(currentAccount.Endpoint, currentAccount.Key, currentAccount.Secret)
 }
 
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
+
 	envs := map[string]string{
-		"CLOUDSTACK_CONFIG": "config",
-		"CLOUDSTACK_REGION": "region",
+		"EXO_CONFIG":  "config",
+		"EXO_ACCOUNT": "account",
```

Comment:
to be consistent, use the full `EXOSCALE` name

Reply:
👍